cmd/kiya/utils.go

```diff
@@ -36,3 +40,26 @@ func promptForYes(message string) bool {
 	yn, _ := reader.ReadString('\n')
 	return strings.HasPrefix(yn, "Y") || strings.HasPrefix(yn, "y")
 }
+
+func shouldPromptForPassword(b backend.Backend) bool {
+	switch b.(type) {
+	case *backend.FileStore:
+		return true
+	default:
+		return false
+	}
+}
+func promptForPassword() []byte {
+	fmt.Println("Enter master password: ")
+	password, err := term.ReadPassword(syscall.Stdin)
+
+	if err != nil {
+		log.Fatal("Error while reading password from standard in", err)
+	}
+
+	err = passwordvalidator.Validate(string(password), 50)
```

Comment:
https://github.com/wagslane/go-password-validator to understand the entropy

Reply:
this could mean existing password are not valid.  this could mean that copy secrets might fail because of this.  Is this checked?

Reply:
Existing passwords not working is very theoretical, as people would have needed to check out this branch and create secrets for the file based profile/backend. 

In that case, it may cause passwords to no longer work indeed.
The case above aside, it will NOT interfere with any other backend besides "file".

Copy works by doing a GET and writing the content to the clipboard. Current implementation states that:

if backend == file backend {
   showPasswordPrompt && checkPasswordStrength
}